easybuild/easyconfigs/g/GeoDict/GeoDict-2024.SP2-GCC-12.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'Tarball'
+
+name = 'GeoDict'
+version = '2024.SP2'
+
+homepage = 'https://www.math2market.com/geodict-software/geodict-applications.html'
+description = """
+The innovative and easy-to-use material simulator GeoDict is the most complete software
+solution for multi-scale 3D image processing, modeling of materials, visualization, material
+property characterization, simulation-based material development, and optimization of
+processes.
+"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = [
+    'https://www.gddownload.de/Releases'
+]
+sources = [
+    '%(name)s2024-3-4-Linux-x86_64-ServicePack2.tar.gz',
+    '%(name)s2024-3-4-Linux-x86_64-ServicePack2-Tools.tar.gz'
+]
+checksums = [
+    {'GeoDict2024-3-4-Linux-x86_64-ServicePack2.tar.gz':
+     '049401063d892eac65696d7d1ed8d4b915c9a81bb13e325cc0657c60ce8aeb28'},
+    {'GeoDict2024-3-4-Linux-x86_64-ServicePack2-Tools.tar.gz':
+     '1b50fb840e8e78c225c748d73eaca60faedd822de33f20c716b1e98b15492eac'},
+]
+
+dependencies = [
+    ('X11', '20230603'),
+    ('MPICH', '4.2.1'),
```

Comment:
It's probably better to define a new version of the `gmpich` toolchain rather than using MPICH as a normal dependency, to align with how we deal with Open MPI and Intel MPI?